overlord/servicestate/quota_control_test.go

```diff
@@ -921,3 +923,97 @@ func (s *quotaControlSuite) TestCreateQuotaCreateQuotaConflict(c *C) {
 	_, err = servicestate.CreateQuota(st, "foo", "", []string{"test-snap2"}, 2*quantity.SizeGiB)
 	c.Assert(err, ErrorMatches, `quota group "foo" has "quota-control" change in progress`)
 }
+
+func (s *quotaControlSuite) TestMemoryCGroupDisabled(c *C) {
+	s.state.Lock()
+	defer s.state.Unlock()
+
+	_memoryCGroupFile := servicestate.CGroupsFilePath
```

Comment:
when you make the change to define the /procs/cgroup file as a variable from dirs package prefixed with dirs.GlobalRootDir, then when dirs.GlobalRootDir is some path like `/tmp/work/foo/test/0`, then all the other tests which rely on quota stuff will start failing, which is okay, the easy thing to do is to create a simple example file with the memory cgroup enabled at /tmp/.../proc/cgroup in the quotaControlSuite.SetUpTest() function. Then here in these two functions you can replace that file with a different file for whatever you need.